lib/matplotlib/tests/test_rcparams.py

```diff
@@ -204,6 +204,15 @@ def test_Issue_1713(tmpdir):
     assert rc.get('timezone') == 'UTC'
 
 
+def test_animation_frame_formats():
+    # Animation frame_format should allow any of the following
+    # if any of these are not allowed, an exception will be raised
```

Comment:
Optionally add another test checking the exception for this case.

Reply:
I can do that, but I think that correctness is implied by an already existing test for `validate_stringlist`.